FixedArray.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  JTFixedArray.swift
+//  
+//  A fixed length array type in Swift.
+//
+//  Created by Ben on 26/05/2016.
+
+import Foundation
+
+struct JTFixedArray<T: Comparable> {
+	private let maxSize: Int
+	private let defaultValue: T!
+	private var array: [T]
+	private (set) var count = 0
```

Comment:
`count` of the _actual_ set data values. So in the initial state the array is "empty" with `["", "", "", ""]` as an example.

I could make it so that the array is completely empty at the beginning though...